common/static/common/js/spec_helpers/discussion_spec_helper.js

```diff
@@ -1,113 +1,121 @@
 /* global DiscussionCourseSettings, DiscussionUtil, DiscussionUser */
-(function () {
+(function() {
     'use strict';
-    this.DiscussionSpecHelper = (function () {
-
+    this.DiscussionSpecHelper = (function() {
         function DiscussionSpecHelper() {
         }
 
-        DiscussionSpecHelper.setUpGlobals = function () {
-            DiscussionUtil.loadRoles({
-                "Moderator": [],
-                "Administrator": [],
-                "Community TA": []
-            });
-            window.$$course_id = "edX/999/test";
-            window.user = new DiscussionUser({
-                username: "test_user",
-                id: "567",
-                upvoted_ids: []
-            });
+        DiscussionSpecHelper.setUpGlobals = function() {
+            DiscussionUtil.loadRoles(DiscussionSpecHelper.getTestRoleInfo());
+            window.$$course_id = 'edX/999/test';
+            window.user = new DiscussionUser(DiscussionSpecHelper.getTestUserInfo());
             return DiscussionUtil.setUser(window.user);
         };
 
-        DiscussionSpecHelper.makeTA = function () {
-            return DiscussionUtil.roleIds["Community TA"].push(parseInt(DiscussionUtil.getUser().id));
+        DiscussionSpecHelper.getTestUserInfo = function() {
+            return {
+                username: 'test_user',
+                id: '567',
+                upvoted_ids: []
+            };
+        };
+
+        DiscussionSpecHelper.getTestRoleInfo = function() {
+            return {
+                Moderator: [],
+                Administrator: [],
+                'Community TA': []
```

Comment:
Yep, [upstream is the same](https://github.com/edx/edx-platform/blob/master/common/static/common/js/spec_helpers/discussion_spec_helper.js#L25-L27).